package.json

```diff
@@ -181,7 +181,7 @@
     "abi-decoder": "^2.4.0",
     "axios": "0.21.1",
     "bignumber.js": "9.0.1",
-    "bnc-onboard": "~1.30.0",
+    "bnc-onboard": "^1.32.0",
```

Comment:
I would recommend to keep the ~ for semver here, as we had problems in past with ^ by introducing breaking changes in minor versions